### ISSUE TITLE
Static coordinate locking for nodes

### ISSUE THREAD

**Issue by knackstedt** [vasturiano/3d-force-graph#468]
**Is your feature request related to a problem? Please describe.**
I'd like to be able to show something like the DAG mode tree graph, where nodes are locked to a vertical axis based on some configuration. I want it to show peer links and self-links. 
It could be as simple as adding an option for DAG to use a node accessor to figure out the offset from the middle/bottom and allow peer and self linking

**Describe the solution you'd like**
I want an option added for something along the lines of nodeYPosition, that can statically set the vertical position of nodes in the graph while still keeping the rest of the force-graph dynamic positioning. 

**Describe alternatives you've considered**
It may be possible by going in the Three Objects in the underlying system and forcing their positioning, but I don't think that's an optimal solution.

**Additional context**
Essentially I want to lock each color of node to a vertical layer and allow connections to loop back to a node and cross to other nodes horizontally. The DAG doesn't allow for that.

![image](https://user-images.githubusercontent.com/29349167/118181204-7672c000-b405-11eb-99ff-7dd1365bbb81.png)

**Comment by vasturiano** [comment]
@knackstedt you can easily fix a node onto a vertical plane by simply adding an `fy: (the y value)` attribute to your nodes.
That's actually what the DAG mode does under the hood.

There's more docs about fixing node here: https://github.com/vasturiano/d3-force-3d#simulation_nodes

**Comment by knackstedt** [comment]
@vasturiano 

I see! I figured I was either looking at the wrong properties or nobody had thought of it before. I managed to get it working with this... your solution is **definitely** far better :)

```
      graph.onEngineTick(() => {
        this.graph.nodeThreeObject(e => { e.y = e.yOffset });
      })
```

Incidentally, is there something that I can toggle to get the FPS of the rendering to show up? 
Context: I'm absolutely abusing your library and WebGL trying to render thousands of nodes and links, want to set a hard limit and optimize it based on the FPS of the renderer.
